source/support.html.erb

```diff
@@ -23,53 +23,40 @@ title: Support
 
       <p>
         GOV.UK PaaS is supported 24 hours a day, seven days a week by the team
-        that builds it.
+        that builds it. Response times and resolutions vary depending on the service status.
       </p>
-
+      
       <p>
-        On this page you can find out about the support we provide during and
-        outside of working hours, and how you can contact us to report a problem
-        or ask for help.
+        You can view the availability and database connectivity of live applications on GOV.UK PaaS on our <a href="https://status.cloud.service.gov.uk/">system status page</a>.
       </p>
 
       <p>
-        You can view the availability and database connectivity of live
-        applications on GOV.UK PaaS on our <a href="https://status.cloud.service.gov.uk/">system status page</a>.
-      </p>
-      
+       
 
       <h2 class="heading-large">Support during working hours</h2>
 
       <p>
-        Our office hours are 9am - 5pm Monday to Friday. During this time we support:
-
+        Our office hours are 9am - 5pm Monday to Friday. You can email us at <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">
+          gov-uk-paas-support@digital.cabinet-office.gov.uk to:
 
       <ul>
-        <li>requests for help using GOV.UK PaaS</li>
-        <li>fixing issues and resolving incidents</li>
+        <li>report issues and incidents</li>
+        <li>get help using GOV.UK PaaS</li>
       </ul>
-
-      <p>
-        We prioritise support for live services over those in development, and
-        fixing issues with the platform over answering user queries. For more
-        information on how we respond to requests, you can look at our platform
-        documentation.
       </p>
 
       <p>
-        For help using GOV.UK PaaS or to report a problem with your service,
-        email us at
-        <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">
-          gov-uk-paas-support@digital.cabinet-office.gov.uk
-        </a>.
+        We prioritise fixing issues with the platform over answering user queries, and support for live services over support for services in development.
+        For more information on how we respond to requests, you can look at our <a href="https://docs.cloud.service.gov.uk/#technical-documentation-for-gov-uk-paas">technical documentation</a>.
       </p>
 
+      <p>
+      
       <h2 class="heading-large">Support outside of working hours</h2>
 
       <p>
-        Between 5pm and 9am Monday to Friday, at weekends, and during Bank
-        Holidays we provide emergency support for critical incidents affecting
-        live production services. A critical incident is where:
+       Between 5pm and 9am on Monday to Friday, at weekends and during Bank Holidays we provide emergency support for critical incidents affecting live production services.
+       A critical incident is where:
       </p>
```

Comment:
Some content (bulleted list) isn't displaying here but it's in the code... I don't know how to check whether there's a problem with it or the display.

Reply:
This view is only showing you the changes and the bulleted list below this isn't something that's changed.

If you click on the View button above it'll show you the entire file after the changes.